test/Controllers/ServicesTest.php

```diff
@@ -0,0 +1,240 @@
+        $responseBody = $this->bodyFromResponse($response, 'create');
+        $this->assertEquals($responseBody, $createServiceBody);
+
+        // Delete test service:
+        $delResponse = $this->services->deleteServiceById(Configuration::$authorization, $responseService->service->id);
+    }
+
+    /**
+     * Test PUT /rervices/{id}
+     * GonebusyLib\Controllers\ServicesController::updateServiceById()
+     */
+    public function testUpdateServiceById() {
+        $createServiceBody = $this->serviceBody('create');
+        $createResponse = $this->services->createService(Configuration::$authorization, $createServiceBody);
+
+        $anotherServiceBody = $this->serviceBody('update');
+
+        // Update the same user:
+        $response = $this->services->updateServiceById(
+            Configuration::$authorization,
+            $createResponse->service->id,
+            $anotherServiceBody);
+
+        // Was it fetched?
+        $this->assertInstanceOf('GonebusyLib\Models\UpdateServiceByIdResponse', $response);
+
+        // Does it have all the new data we sent?
+        $responseBody = $this->bodyFromResponse($response, 'update');
+        $this->assertEquals($responseBody, $anotherServiceBody);
+
+        // Delete test service:
+        $delResponse = $this->services->deleteServiceById(Configuration::$authorization, $createResponse->service->id);
+    }
+
+    /**
+     * Test GET /services
+     * Assumes there's at least 3 Services in the API's database.
+     * GonebusyLib\Controllers\ServicesController::getServices()
+     */
+    public function testGetServices() {
+        $perPage = 3;
+        $response = $this->services->getServices(
+            Configuration::$authorization,
+            $page = 1,
+            $perPage);
+
+        // Was it fetched?
+        $this->assertInstanceOf('GonebusyLib\Models\GetServicesResponse', $response);
+
+        // Did it return an array of 3 services?
+        $this->assertCount($perPage, $response->services);
+        foreach($response->services as $service) {
+            $this->assertInstanceOf('GonebusyLib\Models\GetServicesResponse', $response);
+        }
+    }
+
+    /**
+     * Test GET /services/{id}/available_slots
+     * GonebusyLib\Controllers\ServicesController::getServices()
+     */
+    public function testGetServiceAvailableSlotsById() {
+        $createServiceBody = $this->serviceBody('create');
+        $responseService = $this->services->createService(Configuration::$authorization, $createServiceBody);
+
+        $date = date('Y-m-d'); // Now
+        $response = $this->services->getServiceAvailableSlotsById(
+            Configuration::$authorization,
+            $responseService->service->id,
+            $date//,
+            // $endDate,
+            // $startDate
+        );
+        print_r($response);
+
+        // Was it fetched?
+        $this->assertInstanceOf('GonebusyLib\Models\GetServiceAvailableSlotsByIdResponse', $response);
```

Comment:
@alexagranov I don't see anything else we could assert here but please lmk otherwise. `$response` is returned as:

```sh
GonebusyLib\Models\GetServiceAvailableSlotsByIdResponse Object
(
    [service] => GonebusyLib\Models\EntitiesAvailabilityResponse Object
        (
            [id] => 6312598510
            [ownerId] => 8552697701
            [resources] => Array
                (
                )

        )

)
```